CHANGELOG.md

```diff
@@ -6,6 +6,14 @@
 
 #### New Features
 - Added support for `snowflake.snowpark.testing.assert_dataframe_equal` that is a util function to check the equality of two Snowpark DataFrames.
+- Added support for specifying the following attributes when create `DataFrame.create_or_replace_dynamic_table`:
```

Comment:
Added support for specifying the following parameters to `DataFrame.create_or_replace_dynamic_table()`